drivers/rke/rke_driver.go

```diff
@@ -186,11 +189,17 @@ func (d *Driver) Update(ctx context.Context, clusterInfo *types.ClusterInfo, opt
 
 	certStr := ""
 
+	// Get rotate certificate flag
+	GetRotateCertsOptions(opts, &rkeConfig)
 	dialers, externalFlags := d.getFlags(rkeConfig, stateDir)
 	if err := cmd.ClusterInit(ctx, &rkeConfig, dialers, externalFlags); err != nil {
 		return nil, err
 	}
-	APIURL, caCrt, clientCert, clientKey, certs, err := cmd.ClusterUp(ctx, dialers, externalFlags)
+	if opts.BoolOptions["rotateCertificates"] {
+		APIURL, caCrt, clientCert, clientKey, certs, err = cmd.RebuildClusterWithRotatedCertificates(ctx, dialers, externalFlags)
+	} else {
+		APIURL, caCrt, clientCert, clientKey, certs, err = cmd.ClusterUp(ctx, dialers, externalFlags)
```

Comment:
instead of calling `RebuildClusterWithRotatedCertificates`, see if we can diverge between clusterup and clusterup with cert rotation. This way you can avoid making kontainer-engine changes at all